uds/transport_interface/abstract_can_transport_interface.py

```diff
@@ -0,0 +1,363 @@
+"""Abstract definition of UDS Transport Interface for CAN bus."""
+
+__all__ = ["AbstractCanTransportInterface", "FlowControlGeneratorAlias"]
```

Comment:
`FlowControlGeneratorAlias` probably does not have to be visible outside this file